### ISSUE TITLE
[Paper] nGPT: Normalized Transformer with Representation Learning on the
  Hypersphere

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed-dev#107]
{
  "arxivId": "2410.01131",
  "url": "https://arxiv.org/pdf/2410.01131",
  "title": "nGPT: Normalized Transformer with Representation Learning on the\n  Hypersphere",
  "authors": "Ilya Loshchilov, Cheng-Ping Hsieh, Simeng Sun, Boris Ginsburg",
  "abstract": "We propose a novel neural network architecture, the normalized Transformer\n(nGPT) with representation learning on the hypersphere. In nGPT, all vectors\nforming the embeddings, MLP, attention matrices and hidden states are unit norm\nnormalized. The input stream of tokens travels on the surface of a hypersphere,\nwith each layer contributing a displacement towards the target output\npredictions. These displacements are defined by the MLP and attention blocks,\nwhose vector components also reside on the same hypersphere. Experiments show\nthat nGPT learns much faster, reducing the number of training steps required to\nachieve the same accuracy by a factor of 4 to 20, depending on the sequence\nlength.",
  "timestamp": "2024-12-15T17:43:07.211Z",
  "rating": "novote"
}

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.
